tests/unit/test_create_dataset.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+auth = UsernamePasswordAuth("username", "password")
+unify = Client(auth)
+
+
+@responses.activate
+def test_create_dataset():
+    dataset_config = {
+        "id": "unify://unified-data/v1/datasets/1",
+        "name": "dataset",
+        "keyAttributeNames": ["F1"],
+        "description": "So much data in here!",
+        "externalId": "Dataset created with pubapi",
+    }
+
+    datasets_url = f"http://localhost:9100/api/versioned/v1/datasets"
+    dataset_url = f"http://localhost:9100/api/versioned/v1/datasets/1"
+
+    responses.add(responses.POST, datasets_url, json=dataset_config, status=204)
+    responses.add(responses.GET, dataset_url, json=dataset_config)
+
+    u = unify.create_dataset(dataset_config)
+    p = unify.datasets.by_resource_id("1")
+    assert u.id == p.id
```

Comment:
This is causing tests to fail. I think should be:

```suggestion
    assert u.resource_id == p.resource_id
```

Reply:
@mollysacks you can [run tests locally](https://tamr-unify-python-client.readthedocs.io/en/latest/contributor-guide.html#run-tests) to make sure your PR will pass the automated testing.